# Researchy project yummy :3

In [2]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns

In [3]:
gpp = pd.read_csv('1- global-plastics-production.csv')
sgp = pd.read_csv('2- share-of-global-plastic-waste-emitted-to-the-ocean.csv')
spf = pd.read_csv('3- share-plastic-fate.csv')
mpw = pd.read_csv('4- mismanaged-plastic-waste-per-capita.csv')
#mppd = pd.read_csv('MP_Papers_Dryad.xls')
ppd = pd.read_csv('Plastic_Policies_Dryad.csv')


In [4]:
gpp = pd.read_csv('1- global-plastics-production.csv')
gpp.info()
#There are no null variables in dataset global-plastics-production

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 69 entries, 0 to 68
Data columns (total 4 columns):
 #   Column                                           Non-Null Count  Dtype 
---  ------                                           --------------  ----- 
 0   Entity                                           69 non-null     object
 1   Code                                             69 non-null     object
 2   Year                                             69 non-null     int64 
 3   Annual plastic production between 1950 and 2019  69 non-null     int64 
dtypes: int64(2), object(2)
memory usage: 2.3+ KB


In [5]:
gpp

Entity      Code  Year  Annual plastic production between 1950 and 2019
0   World  OWID_WRL  1950                                          2000000
1   World  OWID_WRL  1951                                          2000000
2   World  OWID_WRL  1952                                          2000000
3   World  OWID_WRL  1953                                          3000000
4   World  OWID_WRL  1954                                          3000000
..    ...       ...   ...                                              ...
64  World  OWID_WRL  2015                                        381000000
65  World  OWID_WRL  2016                                        400050000
66  World  OWID_WRL  2017                                        420052500
67  World  OWID_WRL  2018                                        441055140
68  World  OWID_WRL  2019                                        459746020

[69 rows x 4 columns]

In [7]:
sgp = pd.read_csv('2- share-of-global-plastic-waste-emitted-to-the-ocean.csv')
sgp.info()
# there are 11 null variables in Code

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 170 entries, 0 to 169
Data columns (total 4 columns):
 #   Column                                     Non-Null Count  Dtype  
---  ------                                     --------------  -----  
 0   Entity                                     170 non-null    object 
 1   Code                                       159 non-null    object 
 2   Year                                       170 non-null    int64  
 3   Share of global plastics emitted to ocean  170 non-null    float64
dtypes: float64(1), int64(1), object(2)
memory usage: 5.4+ KB


In [7]:
sgp

Entity Code  Year  Share of global plastics emitted to ocean
0                 Africa  NaN  2019                                   7.989317
1                Albania  ALB  2019                                   0.159782
2                Algeria  DZA  2019                                   0.589510
3                 Angola  AGO  2019                                   0.087804
4    Antigua and Barbuda  ATG  2019                                   0.000204
..                   ...  ...   ...                                        ...
165            Venezuela  VEN  2019                                   0.611359
166              Vietnam  VNM  2019                                   2.881287
167       Western Sahara  ESH  2019                                   0.003880
168                Yemen  YEM  2019                                   0.025729
169             Zimbabwe  ZWE  2019                                   0.000000

[170 rows x 4 columns]

In [8]:
spf = pd.read_csv('3- share-plastic-fate.csv')
spf.info()
#there are 120 null variables in Code

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 200 entries, 0 to 199
Data columns (total 7 columns):
 #   Column                                                      Non-Null Count  Dtype  
---  ------                                                      --------------  -----  
 0   Entity                                                      200 non-null    object 
 1   Code                                                        80 non-null     object 
 2   Year                                                        200 non-null    int64  
 3   Share of waste recycled from total regional waste           200 non-null    float64
 4   Share of waste incinerated from total regional waste        200 non-null    float64
 5   Share of littered and mismanaged from total regional waste  200 non-null    float64
 6   Share of waste landfilled from total regional waste         200 non-null    float64
dtypes: float64(4), int64(1), object(2)
memory usage: 11.1+ KB


In [8]:
spf

Entity      Code  Year   
0    Americas (excl. USA)       NaN  2000  \
1    Americas (excl. USA)       NaN  2001   
2    Americas (excl. USA)       NaN  2002   
3    Americas (excl. USA)       NaN  2003   
4    Americas (excl. USA)       NaN  2004   
..                    ...       ...   ...   
195                 World  OWID_WRL  2015   
196                 World  OWID_WRL  2016   
197                 World  OWID_WRL  2017   
198                 World  OWID_WRL  2018   
199                 World  OWID_WRL  2019   

     Share of waste recycled from total regional waste   
0                                             4.721963  \
1                                             4.972505   
2                                             5.227469   
3                                             5.482511   
4                                             5.740007   
..                                                 ...   
195                                           8.164696   
196                                           8.450203   
197                                           8.747166   
198                                           9.036769   
199                                           9.292344   

     Share of waste incinerated from total regional waste   
0                                             1.220057     \
1                                             1.208672      
2                                             1.197566      
3                                             1.186352      
4                                             1.175283      
..                                                 ...      
195                                          17.625141      
196                                          18.084925      
197                                          18.578430      
198                                          19.083899      
199                                          19.048033      

     Share of littered and mismanaged from total regional waste   
0                                            32.702644           \
1                                            32.651270            
2                                            32.610554            
3                                            32.546440            
4                                            32.480970            
..                                                 ...            
195                                          24.083096            
196                                          23.626896            
197                                          23.199600            
198                                          22.697952            
199                                          22.454480            

     Share of waste landfilled from total regional waste  
0                                            61.355328    
1                                            61.167557    
2                                            60.964405    
3                                            60.784700    
4                                            60.603745    
..                                                 ...    
195                                          50.127070    
196                                          49.837975    
197                                          49.474804    
198                                          49.181380    
199                                          49.205143    

[200 rows x 7 columns]

In [9]:
mpw = pd.read_csv('4- mismanaged-plastic-waste-per-capita.csv')
mpw.info()
#there are 5 null variables in Code

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 165 entries, 0 to 164
Data columns (total 4 columns):
 #   Column                                             Non-Null Count  Dtype  
---  ------                                             --------------  -----  
 0   Entity                                             165 non-null    object 
 1   Code                                               160 non-null    object 
 2   Year                                               165 non-null    int64  
 3   Mismanaged plastic waste per capita (kg per year)  165 non-null    float64
dtypes: float64(1), int64(1), object(2)
memory usage: 5.3+ KB


In [10]:
ppd = pd.read_csv('Plastic_Policies_Dryad.csv')
ppd.info()
#no null variables in Plastic_Policies_Dryad data set

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 155 entries, 0 to 154
Data columns (total 4 columns):
 #   Column   Non-Null Count  Dtype 
---  ------   --------------  ----- 
 0   Country  155 non-null    object
 1   Year     155 non-null    int64 
 2   Level    155 non-null    object
 3   Type     155 non-null    object
dtypes: int64(1), object(3)
memory usage: 5.0+ KB


In [9]:
ppd

Country  Year Level          Type
0       Denmark  1993  levy  plastic bags
1        Bhutan  1999   ban  plastic bags
2    Bangladesh  2002   ban  plastic bags
3       Ireland  2002  levy  plastic bags
4        Taiwan  2002  levy  plastic bags
..          ...   ...   ...           ...
150      Canada  2021   ban          SUPs
151  Costa Rica  2021   ban          SUPs
152       Spain  2021   ban  plastic bags
153    Thailand  2021   ban  plastic bags
154       China  2022   ban  plastic bags

[155 rows x 4 columns]

In [ ]:
#mppd = pd.read_csv('MP_Papers_Dryad.xls')
#mppd.info()

## Data set:

[Platic Pollution](https://www.kaggle.com/datasets/imtkaggleteam/plastic-pollution?select=4-+mismanaged-plastic-waste-per-capita.csv)
[Plastic polloices](https://zenodo.org/records/4321588)
[Plastic Np papers](https://zenodo.org/records/4321588/files/MP_Papers_Dryad.xls?download=1)
### Question
1. Does the country that bans plastic or levy them have an decrease in annual plastic production 
2. Is there a correlation between Mismangement plastic, and annual plastic production. 
3. Does the publish journal help decrease plastic production and plastic waste

## Information 
[Marine Plastic](https://www.iucn.org/resources/issues-brief/marine-plastic-pollution#:~:text=At%20least%2014%20million%20tons,causes%20severe%20injuries%20and%20death.)
[Video: How We Can Keep Plastics Out of Our Ocean | National Geographic](https://www.youtube.com/watch?v=HQTUWK7CM-Y&pp=ygUObWFyaW5lIHBsYXN0aWM%3D)

1.1
The I would be addressing is about Ocean pollution where each year, millions of pounds of plastic and trash enters the ocean. Some devris have sinked into the ocean and eaten by marine animals taht mistake it for food, and some are in circulating coean currents. The reason why we should be interested in this plastic pollution results in damgein to the environemt, to the health of marine aniaml life, and to economic structuers worldwide. Marine trash is surronded by all manufactured products most of them being plastic that end up ion the ocean. litteriong, storm wind, and poor waste management all contribvute to accumaltion of the debris. Common types of plastic itesm are shpping bags ,beverage bottles, cigarette butts, bottle caps, food wrappers, and fishing gear. Plastic waste is problematic as a pollutant because plastic doesn't degrade as much as other materials whcih can take multpy year to decompose.

1.2
The ways I would plan to address the ocean Plastic pollution is caused in three ways: public awareness and education, international cooperation and consumption, and innovative technologies for clean up. For public awareness and education, implement education campaigns to raise awareness about how plastic pollution has an impact on marine ecosystems and human health. By educating people on the importance of reducing plastic use, we will help reduce plastic going into the ocean. 

For international cooperation, it would be necessary to work with other countries and international organizations to develop and implement strategies for addressing ocean plastic pollution. For supporting global agreements and initiatives aimed at reducing the use of plastic. 

The last one is innovative technology for clean-up. It would be to research and develop innovative technologies for ocean cleanup in order to pick up plastic from the ocean in a very efficient way. For example, floating barries and autonomous vessels. When we collaborate with scientists, engineers, and entrepreneurs, we will be able to scale up those operations to solve those problems.

1.3
My current proposed approach that will address this problem, will be international cooperation the reason why is becasue if we get other countries adn international organization to implement staregeis is decrease platic production, and plastic going into the ocean, for example laws that ban certain plastic production, or education people about on how plastic pollution affect our ocean

1.4
How my anaylsis will help the consumer of my analysis, is because the analysis prodvies aims to allow consumer by offering them to understand of ocean plastic pollution and reachable steps the consumers can take to address the issues.